core/sqf/sqenvcom.sh

```diff
@@ -505,23 +505,6 @@ EOF
   APACHE_HIVE_HOME=$HIVE_HOME
   export HIVE_CNF_DIR=$HIVE_HOME/conf
 
```

Comment:
@zellerh 

Hans- Could you comment on removing this bit of code? I just want to make sure it isn't something thats needed for install_local_hadoop.